src/table/internal.tsx

```diff
@@ -33,10 +33,18 @@ import LiveRegion from '../internal/components/live-region';
 import useTableFocusNavigation from './use-table-focus-navigation';
 import { SomeRequired } from '../internal/types';
 import { TableTdElement } from './body-cell/td-element';
+import { useStickyColumns } from './use-sticky-columns';
+import { StickyColumnsContextProvider } from './sticky-columns-context';
+
 type InternalTableProps<T> = SomeRequired<TableProps<T>, 'items' | 'selectedItems' | 'variant'> &
   InternalBaseComponentProps;
 
-const InternalTable = React.forwardRef(
+export interface CellOffsets {
```

Comment:
This can potentially be optimised. The table only uses `cellOffsets.first[0]` from this structure.